### PR TITLE
Issue30/ft social impact section

### PR DESCRIPTION
**Pull request by Beulah-Matt**
# Description
This pull request is for people to see the impact of elewa group
Made a people's section with animated statistics increasing evenly in about 5 seconds

To achieve this, we needed to:

```
- Make a header showing the current numbers
- Improve the UI by allowing for animation as numbers increased on page load.
- Make it visible on the page so users can notice it easily
```

Fixes # [(issue) - [*Example*: Fixes # ([S4Y-400](https://www.w3.org/Provider/Style/dummy.html), [S4Y-100](https://www.w3.org/Provider/Style/dummy.html))](https://github.com/italanta/elewa-group/issues/30)](https://github.com/italanta/elewa-group/issues/30)

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# Screenshot

finalfullscreen

finalmobile

#Video

https://user-images.githubusercontent.com/109944021/220607910-81301b11-a807-46b2-bae7-ecf476a3f733.mp4

# How Has This Been Tested?

I inspected the component for both full view and on mobile view with a media query of max 600px

# Checklist:
- [yes ] My code follows the style guidelines of this project
- [yes ] My changes generate no new warnings
- [ yes] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Beulah-Matt** [comment]
@IanOdhiambo9 kindly review this request.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Looks good as of now 👍 . Please link a video showing how the increment works.

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 1]
```diff
@@ -3,9 +3,17 @@ import { CommonModule } from '@angular/common';
 import { SocialImpactPageComponent } from './pages/social-impact-page/social-impact-page.component';
 
 import { SocialImpactRoutingModule } from './social-impact.routing';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:5]: RoutingModule import comes last, move this to the end.

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 2]
```diff
@@ -3,9 +3,17 @@ import { CommonModule } from '@angular/common';
...
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { ElewaGroupImageAndTextBannerComponent } from 'libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:7]: RoutingModule import comes last, move this to the end.

Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:7]: Import from the aliased path

**Pull request closed without merging**